src/main/java/com/hazelcast/commandline/member/ProcessStore.java

```diff
@@ -75,21 +80,20 @@ void save(HazelcastProcess process)
         if (Files.size(path) == 0) {
             return processes;
         }
-        try (FileInputStream fileInputStream = new FileInputStream(instancesFilePath);
-             ObjectInputStream input = new ObjectInputStream(fileInputStream)) {
-            processes = (Map<String, HazelcastProcess>) input.readObject();
-        } catch (ClassNotFoundException cnfe) {
-            throw new HazelcastException(cnfe.getMessage(), cnfe);
+        try (FileInputStream fileInputStream = new FileInputStream(instancesFilePath)) {
+            Input input = new Input(fileInputStream);
+            processes = kryo.readObject(input, HashMap.class);
+            input.close();
```

Comment:
Do we need `input.close` as Input is AutoCloseable
https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html
https://github.com/EsotericSoftware/kryo/blob/master/src/com/esotericsoftware/kryo/io/Input.java#L33

Reply:
Yes, we don't need it. I will also change the output part with try-with-resources.